README.md

```diff
@@ -30,22 +30,27 @@ please [submit an issue][issues] or [mail us][email].
     Click on "Create repository from template". You will be redirected to
     your new copy of the workshop template repository.
 
-5.  Go to the the workshop repository that is just created. Click on "Settings"
-    tab. In the left sidebar, click on "Pages". Under "GitHub Pages", use the
-    _gh-pages_ branch from drop-down menu and press "save". For more information,
-    see [GitHub
-    documentation](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site). 
+4.  Go to the the workshop repository that is just created. 
+    Manually run the 'Build and deploy Jekyll site to GitHub Pages' github action.
+    See instructions for manually triggering workflows [here][manually-run-workflow]
```

Comment:
It would be even nicer if https://github.com/actions/toolkit/issues/1054 was implemented, than we can also trigger this on repository instantiation.

Reply:
@svenvanderburg What you are trying to do is beautiful 😁 and it can be done:
https://github.com/jlumbroso/hugo-theme-bootstrap-skeleton

It took me a lot of trial and error to get it just right with the sequencing of triggers and actions. Although I may be wrong, I don't think it retriggers afterwards (but if it does, I think there are easy fixes).

Reply:
Literally, this was one of the last trial runs, when I finally got it all right:
https://github.com/jlumbroso/test61

Reply:
@jlumbroso thanks for helping out! I don't think it works, see my comment here: https://github.com/actions/toolkit/issues/1054#issuecomment-1245115705